Sources/ForageSDK/Component/ForagePINTextField/Vault/BtPINTextField.swift

```diff
@@ -152,7 +152,9 @@ class BasisTheoryTextFieldWrapper: UIView, VaultWrapper {
     // MARK: - Public API
     
     func clearText() {
-        textField.text = ""
+        DispatchQueue.main.async {
```

Comment:
Did you test that this crashed before and now this solves the issue? Just making sure we don't introduce a new issue

Reply:
yep, tested!

removing the `DispatchQueue.main.async` causes this to crash:

<img width="691" alt="image" src="https://github.com/teamforage/forage-ios-sdk/assets/32694765/1e1f1cb9-64a8-4f6f-bd5f-ac447f83258a">

inserting it makes it not crash
Also, nesting `DispatchQueue.main.async` is also safe/works, which means if GoPuff wrapped the call to `clearText` in their own `DispatchQueue.main.async`, it would also work 😄 

```swift
        DispatchQueue.main.async {
            DispatchQueue.main.async {
                self.textField.text = ""
            }
        }
```